robolectric/src/test/java/org/robolectric/shadows/SQLiteDatabaseTest.java

```diff
@@ -910,4 +915,16 @@ public void shouldGetBlobFromString() {
         data.moveToFirst();
         assertThat(data.getBlob(0)).isEqualTo(values.getAsString("first_column").getBytes());
     }
+
+    /////////////////////
+
+    private SQLiteDatabase openOrCreateDatabase(String name) {
```

Comment:
Not super straightforward; `android.database.sqlite.SQLiteException: Cannot prepare statement, base error code: 1`, whatever that means.